bindings/python/pymongoarrow/errors.py

```diff
@@ -33,18 +33,14 @@ def __init__(self, details):
     def details(self):
         """Details for the error.
 
-        It is a dictionary of key-value pairs giving diagnostic information about what went wrong. To see the entire dictionary simply use `print(awe.details)`.
```

Comment:
Perhaps add a note that if the error was triggered by a `PyMongoError` then that error will be available on the `__cause__` attribute?  I wasn't aware of that feature.